app.js

```diff
@@ -2,7 +2,7 @@
 // SPDX-License-Identifier: MIT
 
 const express = require('express')
-const logger = require('morgan')
+const morgan = require('morgan')
```

Comment:
We could also consider installing 
var appRoot = require('app-root-path'); 
very useful when specifying paths to files in Node.js code.